lib/index.js

```diff
@@ -60,13 +60,31 @@ const getResultLabel = (e, pageSize) =>
 const getLabelWithdrawnReason = (value) =>
   constants.WITHRAWN_REASONS.find((e) => e.value === value)?.label || value;
 
+function canUpdateStatus(newYoung, oldYoung) {
+  if (!newYoung || !oldYoung) return true;
+  const allStatus = ["status", "statusPhase1", "statusPhase2", "statusPhase3", "statusMilitaryPreparationFiles", "statusPhase2Contract"];
+  if (!allStatus.filter((s) => newYoung[s] !== oldYoung[s]).length) return true;
+
+  const youngStatus = newYoung.status === "VALIDATED" && oldYoung.status !== "VALIDATED";
+  const youngStatusPhase1 = newYoung.statusPhase1 === "DONE" && oldYoung.statusPhase1 !== "DONE";
+  const youngStatusPhase2 = newYoung.statusPhase2 === "VALIDATED" && oldYoung.statusPhase2 !== "VALIDATED";
+  const youngStatusPhase3 = newYoung.statusPhase3 === "VALIDATED" && oldYoung.statusPhase3 !== "VALIDATED";
+  const youngStatusMilitaryPrepFiles = newYoung.statusMilitaryPreparationFiles === "VALIDATED" && oldYoung.statusMilitaryPreparationFiles !== "VALIDATED";
+  const youngStatusPhase2Contract = newYoung.statusPhase2Contract === "VALIDATED" && oldYoung.statusPhase2Contract !== "VALIDATED";
+
+  const authorized = youngStatus || youngStatusPhase1 || youngStatusPhase2 || youngStatusPhase3 || youngStatusMilitaryPrepFiles || youngStatusPhase2Contract;
+
+  return !authorized;
```

Comment:
mmmh je crois que c'est encore a l'envers non ?

`authorized` est a `true` si l'un des champs est en train d'être modifié vers `VALIDATED`
pour rester logique dans la nomenclature j'aurais mis ca
```suggestion
  const notAuthorized = youngStatus || youngStatusPhase1 || youngStatusPhase2 || youngStatusPhase3 || youngStatusMilitaryPrepFiles || youngStatusPhase2Contract;

  return !notAuthorized;
```